pkg/csi/cinder/utils.go

```diff
@@ -57,13 +57,17 @@ func NewIdentityServer(d *Driver) *identityServer {
 	}
 }
 
-func NewNodeServer(d *Driver, mount mount.IMount, metadata metadata.IMetadata, cloud openstack.IOpenStack, topologies map[string]string) *nodeServer {
+func NewNodeServer(d *Driver, mount mount.IMount, metadata metadata.IMetadata, opts openstack.BlockStorageOpts, topologies map[string]string) *nodeServer {
+	if opts.NodeVolumeAttachLimit < 0 || opts.NodeVolumeAttachLimit > maxVolumesPerNode {
+		opts.NodeVolumeAttachLimit = maxVolumesPerNode
```

Comment:
add a log with all opts vars for better visibility

Reply:
AuthOpts does not have access to openstack.BlockStorageOpts opts. Better to add reach logging here https://github.com/kubernetes/cloud-provider-openstack/pull/2652

Reply:
This was just an example, that we need to print a config before starting a service

Reply:
We removed the CLI arguments, so no code was changed. We can improve the extra logging in another PR.